packages/transformers/css/src/CSSTransformer.js

```diff
@@ -83,12 +83,16 @@ export default (new Transformer({
             symbols: new Map([['*', {local: '*', isWeak: true, loc}]]),
           });
         } else if (dep.type === 'url') {
-          asset.addURLDependency(dep.url, {
-            loc,
-            meta: {
-              placeholder: dep.placeholder,
+          asset.addURLDependency(
+            // when URL is escaped, parcel resolver cannot resolve URL, so we need to unescape it in transformer
+            dep.url.replace(/\\"/g, '"').replace(/\\'/g, "'"),
```

Comment:
I don't think this is necessary. The url will already be unescaped before it is passed to us.